src/server/helpers/middleware.ts

```diff
@@ -240,6 +244,14 @@ export function checkValidRevisionId(req: $Request, res: $Response, next: NextFu
 	return next();
 }
 
+export function checkValidRelationshipTypeId(req: $Request, res: $Response, next: NextFunction, id: string) {
+	const idToNumber = _.toNumber(id);
+	if (!_.isInteger(idToNumber) || (_.isInteger(idToNumber) && idToNumber <= 0)) {
```

Comment:
```suggestion
	if (!_.isInteger(idToNumber) || idToNumber <= 0)) {
```
Can be optimized thanks to short-circuit evaluation.